### DISCUSSION TITLE
which version do i use for manjaro

### DISCUSSION THREAD

**Discussion by Barbarossa3** [FreeRDP/FreeRDP#7085] [General]
hi i see that there is 4 version : 
    Ubuntu: https://ci.freerdp.com/job/freerdp-nightly-binaries/
    Debian: https://ci.freerdp.com/job/freerdp-nightly-binaries/
    Fedora: https://ci.freerdp.com/job/freerdp-nightly-rpm-mock/
    OpenSUSE: https://ci.freerdp.com/job/freerdp-nightly-rpm-build/
which one work for Manjaro Kde ?

**Comment by akallabeth** [comment]
We do not provide binaries for manjaro or arch, but you can:
1. Compile the source as described in https://github.com/FreeRDP/FreeRDP/wiki/Compilation
2. Install from AUR (I assume manjaro has that available too?)
